src/styles/less/ParentNavTopBar.less

```diff
@@ -0,0 +1,52 @@
+.parent-nav-top-bar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1% 7%;
+    margin-bottom: 2%;
+    border-bottom: 1px solid gray;
+    background-color: @turquoise-flat;
+    .navbar-logo {
+        font-family: Bangers;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 42px;
+        line-height: 45px;
+        letter-spacing: 0.06em;
+        color: @white;
+        text-shadow:
+            -1.5px -1.5px 0 #000,
+            1.5px -1.5px 0 #000,
+            -1.5px 1.5px 0 #000,
+            1.5px 1.5px 0 #000;
+    }
+    .welcome-back-msg {
+        font-weight: 500;
+        font-size: 0.9rem;
+        margin-right: 12px;
+        letter-spacing: 0;
+        min-width: fit-content;
+    }
+    .parent-avatar {
+        box-sizing: border-box;
+        background: rgba(255, 255, 255, 0.95);
+        border: 0.5px solid #7B8794;
+        border-radius: 100px;
+        img {
+            width: 31px;
+            height: 31px;
+            clip-path: circle();
+        }
```

Comment:
Nice use of clip-path - I hadn't seen that before - is the avatar not a circle already due to the border-radius of the frame being set @ 100px?

Reply:
Thanks!. This is a really good point to raise. AFAIK (from my experience), `border-radius` does not affect SVGs in the same manner that it does other images. With that being said, since I am using an `img` element here, it might actually work!..